src/combobox/tests/unit/ComboBox.ts

```diff
@@ -52,7 +52,11 @@ const getExpectedControls = function(widget: Harness<ComboBox>, useTestPropertie
 	}, [
 		w(TextInput, <any> {
 			key: 'textinput',
-			controls: <any> compareId,
+			aria: {
+				activedescendant: <any> compareId,
```

Comment:
are these `<any>` needed?

Reply:
unfortunately yeah, but should probably be `as any`